ext/fetch/22_http_client.js

```diff
@@ -33,9 +34,14 @@ class HttpClient {
   constructor(rid) {
     this.rid = rid;
   }
+
   close() {
     core.close(this.rid);
   }
+
+  [SymbolDispose]() {
+    this.close();
+  }
```

Comment:
Shouldn't this be `tryClose` so it doesn't throw if someone explicitly calls `close` beforehand?

Reply:
It should! I fixed that and added a test case in https://github.com/denoland/deno/pull/21453/commits/77e0ef2bca5824c1048c49e1975a00e77bdc81a0